core/src/version.rs

```diff
@@ -29,6 +29,13 @@ pub const SPOTIFY_MOBILE_VERSION: &str = "8.6.84";
 pub const FALLBACK_USER_AGENT: &str = "Spotify/117300517 Linux/0 (librespot)";
 
 pub fn spotify_version() -> String {
+    match std::env::consts::OS {
+        "android" | "ios" => SPOTIFY_MOBILE_VERSION.to_owned(),
+        _ => SPOTIFY_VERSION.to_string(),
+    }
+}
+
+pub fn spotify_semantic_version() -> String {
     match std::env::consts::OS {
         "android" | "ios" => SPOTIFY_MOBILE_VERSION.to_owned(),
```

Comment:
I'm not entirely sure which version notation mobile uses, as I currently am not easily able to intercept Spotify communication on mobile. For now I've kept the behavior the same as before d881f46ce744da452417d513f35dd0d070c16509.